site/app/templates/admin/users/StudentList.twig

```diff
@@ -21,14 +21,15 @@
                     <th>Rotating Section</th>
                     <th>UTC Offset / Time Zone</th>
                     <th>View Grades</th>
+                    <th>Registration Type</th>
                     <th>Edit Student</th>
                     <th>Delete Student</th>
                 </tr>
             </thead>
             {% for registration, students in sections %}
                 <tbody id="section-{{ registration }}">
                     <tr class="info">
-                        <th class="section-break" colspan="10">Students Enrolled in Registration Section {{ registration }}</td>
```

Comment:
i feel like this value has been changed a few times. Is there any way to set it to a value? Like `{{ header|length }}` or something passed in from the View? Not necessary but could save someone from future grief.